drivers/blktap2.py

```diff
@@ -840,9 +835,8 @@ def launch_on_tap(cls, blktap, path, _type, options):
 
         except TapCtl.CommandFailure as ctl:
             util.logException(ctl)
-            if ('/dev/xapi/cd/' in path and
-                    'status' in ctl.info and
-                    ctl.info['status'] == 123):  # ENOMEDIUM (No medium found)
+            if (('/dev/xapi/cd/' in path or path.startswith('/dev/sr')) and
```

Comment:
I know it was the existing behaviour, but surely that should be `path.startswith('/dev/xapi/cd')` too ?